bazel/utils/remote.bzl

```diff
@@ -133,69 +133,93 @@ directory that a wrapper requires by default to be used.
     },
 )
 
-_common_attrs = {
-    "wrapper_opts": attr.string_list(
+_known_attributes = [
+    ("wrapper_opts", attr.string_list, dict(
         default = [],
         doc = "Command line options to supply to the wrapper, if any",
-    ),
-    "target": attr.label(
-        executable = True,
-        cfg = "host",
-        mandatory = True,
-        doc = "Target to execute on the remote machine",
-    ),
-    "target_opts": attr.string_list(
+    )),
+    ("target_opts", attr.string_list, dict(
         default = [],
         doc = "Additional command line options to pass to the target on the remote machine",
-    ),
-    "noop": attr.label(
+    )),
+    ("noop", attr.label, dict(
         executable = True,
         cfg = "host",
         default = "@enkit//bazel/utils/remote:noop",
         doc = "If this target is used as wrapper or as a target, it is consider a noop operation. Useful when target or wrapper as specified via label_flag()",
-    ),
-    "rsync_bin": attr.string(
+    )),
+    ("rsync_cmd", attr.string, dict(
```

Comment:
Was thinking about the time it would take to make large copies.. instead of just assuming rsync, we can take a copy_cmd and copy_opts and allow a tool override to use rsync, scp, pdsh cp, mpifilutils, foobar etc later. Perhaps v3 enhancement.

Reply:
let me rename the attributes, I think in practice that's already the case.

Reply:
I spoke too soon: the current shell script relies on a *file* listing all the files to be copied. So it passes `--files-from` to rsync. This is easy to change, simple template substitution. But unless you feel strongly, I'd rather to it in a separate PR.

Reply:
I don't, yeah let's do that in a follow up PR